src/sentry.rs

```diff
@@ -0,0 +1,95 @@
+use std::borrow::Cow;
+
+use chrono::Utc;
+use colored::Colorize as _;
+
+const RELEASE: &str = concat!(env!("CARGO_CRATE_NAME"), "_", env!("CARGO_PKG_VERSION"));
```

Comment:
This is subject for another PR, but maybe we can read `pyproject.toml` in `build.rs`, and update `CARGO_PKG_VERSION` from there? This would avoid parsing it on every command invocation

Reply:
yeah would be great! i mean we could also update the pyproject.toml in the build.rs, so that they're always in sync too

Reply:
i'll handle this in a PR real quick that i'm doing for deps (:

Reply:
My code is using `manifest_version()` now instead 🙂